src/panels.h

```diff
@@ -100,6 +101,10 @@ std::pair<std::string, nc_color> safe_mode_text_color( const bool classic_mode )
 std::pair<std::string, nc_color> wind_text_color( const Character &u );
 std::pair<std::string, nc_color> weather_text_color( const Character &u );
 
+// Get visible threats by cardinal direction - Already colorized
+std::string colorized_compass_text_color( const cardinal_direction dir, int width );
+std::string colorized_compass_legend_text_color( int width, int height );
```

Comment:
```suggestion
std::string colorized_compass_text( const cardinal_direction dir, int width );
std::string colorized_compass_legend_text( int width, int height );
```

Sorry, one last nitpick - the `_text_color` suffix naming convention was intended for the ones returning a `std::pair<std::string, nc_color>` (both text and color).

The ones that return just a `std::string` don't follow a particular pattern so much, but for these pre-colorized ones I'm leaning toward `colorized_xxx_text`, hence the suggestion here. I went with `colorized_bodypart_status_text` for the recently merged bodypart status widgets, and will probably do `colorized_overmap_text` for the upcoming overmap one, so would like to set a good precent early if possible. Thanks!

Reply:
Cheers! I folded those name changes into the previous commit.